Dev

divishinating opened the pull request:
Not sure why you're trying to merge all these commits into my personal fork. I think you meant to put these in your own and somehow ended up here?

unsoluble-test committed:
Fix stale text display logic

unsoluble-test committed:
Simplified stale text

unsoluble-test committed:
Fix stale CSS

MilosKozak committed:
Fix COB calculation in reports

MilosKozak committed:
speedup rendering COB and fix COB=0

MilosKozak committed:
speedup rendering IOB

MilosKozak committed:
fix reloading of missing data

MilosKozak committed:
Merge pull request #2495 from MilosKozak/dev

IOB & COB rendering speedup

MilosKozak committed:
change logig of showing profile link to OR

MilosKozak committed:
Merge pull request #2496 from MilosKozak/dev

change logic of showing profile link to OR

MilosKozak committed:
netIOB report

MilosKozak committed:
Merge pull request #2499 from MilosKozak/dev

netIOB report

MilosKozak committed:
added counts to netIOB report

MilosKozak committed:
Merge pull request #2501 from MilosKozak/dev

added counts to netIOB report

MilosKozak committed:
do not count == 0

MilosKozak committed:
Merge pull request #2502 from MilosKozak/dev

do not count == 0

jasoncalabrese committed:
merge in dev,  fix conflicts in .travis.yml and package.json

jasoncalabrese committed:
use node 6.10.0

jasoncalabrese committed:
set travis to use node 6

jasoncalabrese committed:
fix bug where we didn't wait for all the upserts to complete if an array was posted

jasoncalabrese committed:
undo previous commit

jasoncalabrese committed:
instead of using the data-loaded event, just go back to db directly

jasoncalabrese committed:
try async.forEach again so a post of a treatment array waits for all the upserts

jasoncalabrese committed:
add some debugging to see why count sometimes doesn't match

jasoncalabrese committed:
maybe async.eachSeries will help

jasoncalabrese committed:
resolve conflicts

jasoncalabrese committed:
resolve conflicts

jasoncalabrese committed:
Merge branch 'master' into dev

jasoncalabrese committed:
Merge branch 'dev' into wip/upgrade

1417836931 committed:
Add Simplified Chinese and some Traditional Chinese language. (#2439)

* Update language.js

* Added note about using the same Amazon account as you use on your Alexa app

Initially created my skill using a developer account separate from the app I use to manage Alexa. It wouldn't show up obviously.

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Update language.js

* Remove some period in Chinese

* Update language.js

jasoncalabrese committed:
resolve conflicts

jasoncalabrese committed:
Merge pull request #2430 from nightscout/wip/init-fixes

Wip/init fixes

jasoncalabrese committed:
Merge branch 'dev' into wip/upgrade

gcastellanos committed:
Accessibility improvements - Links and regions on index (#2397)

* added accessibility to the un-named links with the aria-label as well as marked a region as a live region to read the changing portions

* removing script reference from index

* getting rid of script

* accessibility inline script

* Added aria-labels and other accessibility improvements to make site more descriptive to screen readers and other assistive technologies

* labeling blood sugar region and settings form with aria

* Delete output.txt

unsoluble-test committed:
Re-added missing favicon.ico

jasoncalabrese committed:
Merge pull request #2464 from unsoluble/clock-colors-and-trends

Clock colors and trends

jasoncalabrese committed:
bump version to 0.9.3-dev-20170318 after merging several PRs, also merged master back to dev

jasoncalabrese committed:
Merge branch 'dev' into wip/upgrade

oskarpearson committed:
docker: use nodejs 6.10.0

- Pull Node version from .nvmrc
- Migrate passwords out of .travis.yml

oskarpearson committed:
Docker: don't apt-get install npm

As we're using the node:6.10.0 container as a base, we don't need to install
nodejs or nodejs-legacy or npm any longer.

We do still apply security upgrades, but we don't need to remove the apt cache
since the node:6.10.0 container is based on a container that includes
https://github.com/docker/docker/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105

jasoncalabrese committed:
add initial heap dump endpoint if enabled

jasoncalabrese committed:
Merge branch 'wip/upgrade' of github.com:nightscout/cgm-remote-monitor into wip/upgrade

jasoncalabrese committed:
replace all :'s

unsoluble-test committed:
Add viewport scaling & centering

Commit:
fix usage section link

MilosKozak committed:
profiles report

MilosKozak committed:
Merge pull request #2524 from unsoluble/bw-clock-css-update

Add viewport scaling & centering to clock view

MilosKozak committed:
fix entering decimal numbers

MilosKozak committed:
Merge pull request #2541 from MilosKozak/dev

profiles report

jasoncalabrese committed:
Merge pull request #2526 from ollieglass/fix_readme_link

fix usage section link

jasoncalabrese committed:
remove clipping path that wasn't needed and caused problems on iOS

jasoncalabrese committed:
bump version to 0.10.0-dev-20170423 before merging to dev

jasoncalabrese committed:
Merge pull request #2071 from nightscout/wip/upgrade

target node 6.10.0 and upgrade benv and some other dependencies

bogdangorescu committed:
RO - update 05.05.2017

RO language

MilosKozak committed:
filter by hours in distribution

jasoncalabrese committed:
bump benv to version 3.3.0 to fix issue with updated jsdom

jasoncalabrese committed:
don't try toggling the drawer durring the test

jasoncalabrese committed:
Merge pull request #2609 from nightscout/wip/benv-upgrade

bump benv to version 3.3.0 to fix issue with updated jsdom

MilosKozak committed:
Merge pull request #2608 from MilosKozak/dev

filter by hours in distribution

MilosKozak committed:
Merge pull request #2570 from bogdangorescu/patch-21

RO - update 05.05.2017

MilosKozak committed:
process durations only on clients

apanasef committed:
russian translation update

added missed translations, minor corrections

MilosKozak committed:
Merge pull request #2614 from apanasef/patch-17

russian translation update

MilosKozak committed:
send mills of removed treatments too

sulkaharo committed:
Variability reporting

sulkaharo committed:
explanations, filter skipped readings

sulkaharo committed:
mmol support

sulkaharo committed:
translations

sulkaharo committed:
Merge branch 'dev' into variability

sulkaharo committed:
Added localization & missing keys

sulkaharo committed:
Oops now using correct place to fetch units

sulkaharo committed:
Removed logging

sulkaharo committed:
Make tests happy

sulkaharo committed:
Merge pull request #2628 from sulkaharo/variability

Variability reporting, distribution reporting fixes

sulkaharo committed:
Fixed a bug with calculating the metrics when CGM events were skipped

sulkaharo committed:
Merge pull request #2630 from sulkaharo/variability

Fixed a bug with calculating the metrics when CGM events were skipped

ps2 committed:
Track last fetched sensor values, and attempt to backfill

scottleibrand committed:
display floating carbs / UAM purple line (#2564)

MilosKozak committed:
Pump Battery Change event load

MilosKozak committed:
Merge remote-tracking branch 'upstream/dev' into dev

MilosKozak committed:
fix report test

MilosKozak committed:
Merge pull request #2637 from MilosKozak/dev

updates for AAPS

ps2 committed:
Handle test passing in null glucose

ps2 committed:
Update example start command to allow spaces in vars

ps2 committed:
Track objects created in upsert, so we can generate response with ids for clients

jasoncalabrese committed:
Merge pull request #2653 from ps2/respond-with-created-treatments

Track objects created in upsert, so we can generate response with ids…

jasoncalabrese committed:
Merge pull request #2652 from ps2/env-spaces

Update example start command to allow spaces in vars

jasoncalabrese committed:
Merge pull request #2635 from ps2/backfill

Track last fetched sensor values, and attempt to backfill

ps2 committed:
Add food type string to carb entry on graph, and show Absorption Time in tooltip

ps2 committed:
fix incorrect variable name

jasoncalabrese committed:
Merge pull request #2657 from ps2/food-type

Add food type string to carb entry on graph.

sulkaharo committed:
Update package.json to run node 6.11.1

sulkaharo committed:
Merge pull request #2681 from sulkaharo/dev

Update package.json to run node 6.11.1

sulkaharo committed:
Merge branch 'dev' into wip/if-modified-since

Conflicts:
	lib/api/treatments/index.js
	package.json

jasoncalabrese committed:
update to node 8.1.4
also adding new npm package-lock.json to repo as recommended by https://docs.npmjs.com/files/package-lock.json

jasoncalabrese committed:
bump version to 0.10.0-dev-20170716

jasoncalabrese committed:
Merge pull request #2688 from nightscout/wip/node-814

update to node 8.1.4

sulkaharo committed:
Updated package.json to 8.1.4, updated modules

sulkaharo committed:
Merge commit '25b9bce53f81e1ad2bed278c6982bb1eef125d84' into wip/if-modified-since

# Conflicts:
#	package-lock.json

unsoluble committed:
Merge pull request #7 from nightscout/dev

Dev update

unsoluble-test committed:
Fix clock trend arrow alignment in iOS7/8

sulkaharo committed:
Merge pull request #2393 from sulkaharo/wip/if-modified-since

If-Modified-Since support for API

tanja3981 committed:
alexa translations

tanja3981 committed:
alexa translations: iob

tanja3981 committed:
alexa translations: iob

tanja3981 committed:
alexa translations: iob

tanja3981 committed:
alexa translations: default en

sulkaharo committed:
Fix pump suspend preferences

sulkaharo committed:
Merge pull request #2718 from sulkaharo/dev

Fix pump suspend preferences

MilosKozak committed:
add missing indexes for nsclient

MilosKozak committed:
Merge pull request #2727 from MilosKozak/dev

add missing indexes for nsclient

MilosKozak committed:
proper sort in calibration plugin

MilosKozak committed:
Merge pull request #2733 from MilosKozak/dev

proper sort in calibration plugin

sulkaharo committed:
Data cleaning pass 1

sulkaharo committed:
* Glucose data smoothing / patching before calculating variability to reduce the effects of CGM noise
* Added calculation for GVI, PGS and Total Absolute Change calculation
* Refactored the server to use Webpack for the bundle generation instead of Browserify (since browserify-express seems to not work with Node 8 and is not maintained)
* Improved caching & compression of JS and CSS resources
* If NODE_ENV="development”, don’t cache resources for more than 10 seconds & generate a report about Bundle size

sulkaharo committed:
Run packaging in post install instead of runtime

sulkaharo committed:
CSS bundling

sulkaharo committed:
Fix templates that linked to old bundle.js location

sulkaharo committed:
Removing Bower part 1

sulkaharo committed:
Flot plugins

sulkaharo committed:
jQuery packaging has changed, adding these to the repo

sulkaharo committed:
Generate source map, use a small font subset instead of loading complex CSS / font set from Google

sulkaharo committed:
Added missing icons, changed how jQuery and jQuery-UI is packaged

sulkaharo committed:
Bower is gone

sulkaharo committed:
Remove Crossfilter (needed by /reports/compare, which is not supported right now)

sulkaharo committed:
Oops forgot package-lock.json

sulkaharo committed:
Revert back to using Open Sans, but embedded

sulkaharo committed:
Re-enabled Ubuntu to fix the layouts. Icon font embedded.

sulkaharo committed:
Update cache busters

sulkaharo committed:
Added variability measure explanations to reporting

sulkaharo committed:
Finnish localization

sulkaharo committed:
Fix battery icon, some tests

sulkaharo committed:
Added app cache manifest to speed things further

sulkaharo committed:
Launch PNG was missing

sulkaharo committed:
Return 404 on app cache request when in development mode, added bold weight Ubuntu

sulkaharo committed:
Formatting improvements to app.js

sulkaharo committed:
Merge pull request #2736 from sulkaharo/wip/variability

Variability reporting, Webpack bundling

sulkaharo committed:
Added sync-exec in an attempt to appease Azure

sulkaharo committed:
Merge pull request #2759 from sulkaharo/wip/variability

Added sync-exec in an attempt to appease Azure

sulkaharo committed:
Added --scripts-prepend-node-path to Azure deploy script

sulkaharo committed:
Merge pull request #2760 from sulkaharo/dev

Added --scripts-prepend-node-path to Azure deploy script

hichameyessou committed:
IT Language

apanasef committed:
update language.js

update Russian translation

sulkaharo committed:
Merge pull request #2762 from apanasef/patch-18

update language.js

sulkaharo committed:
Merge branch 'dev' into patch-1

sulkaharo committed:
Merge pull request #2761 from hichameyessou/patch-1

IT Language

tanja3981 committed:
Merge branch 'ns_dev' into alexaTranslations

bogdangorescu committed:
RO update August, 27th - 2017

RO update

sulkaharo committed:
Trigger app on DOM ready event (as recommended by HTML5 spec). Detect if app is completely offline, auto-reload if needed.

sulkaharo committed:
Better looking loader screen, prevent CSS blocking

sulkaharo committed:
Use ejs to render the app cache and cache buster tokens

sulkaharo committed:
Forgot cache buster generator script

sulkaharo committed:
Image had ugly artifacts

sulkaharo committed:
Upgrade Travis to build with Node 8.4, update package.json to 8.4

sulkaharo committed:
Merge branch 'dev' into wip/onlinestatus

sulkaharo committed:
Doh, 8.1.4 is the latest version of Node available in Azure

sulkaharo committed:
Added note on configuring Azure to README

sulkaharo committed:
Separate messages for loading the client and connecting to server

sulkaharo committed:
Hide loader on auth fail

PieterGit committed:
Support both node 8.1.4 (Azure) and node 8.4.0 (Heroku).

Make sure travis builds / tests both versions. Got this syntax from: https://github.com/SAP/chevrotain/commit/cedbffd3261a4f2bfe142ea11d62b9af4fd8d0cb and https://docs.microsoft.com/en-us/azure/nodejs-specify-node-version-azure-apps

sulkaharo committed:
Unit tests fixed

sulkaharo committed:
Increasing test timeout significantly due to Travis fails

sulkaharo committed:
And add the longer timeout to Travis, too…

sulkaharo committed:
Now it works

sulkaharo committed:
Fixed Coverage running parameter order

sulkaharo committed:
Report tests takes over 40 seconds to run in Travis, timeout increased

sulkaharo committed:
Merge pull request #2775 from sulkaharo/dev

Unit tests fixed

sulkaharo committed:
Merge commit '035b3aef7418d1dd4d2a2ffec86e3363dc7fe1b2' into wip/onlinestatus

sulkaharo committed:
More test fixes

sulkaharo committed:
Increasing Travis timeouts ridiculously, we need to do something to this

sulkaharo committed:
Merge pull request #2767 from bogdangorescu/patch-22

RO update August, 27th - 2017

sulkaharo committed:
Merge branch 'dev' into wip/onlinestatus

tanja3981 committed:
Merge branch 'dev' of https://github.com/nightscout/cgm-remote-monitor into alexaTranslations

jasoncalabrese committed:
minor css fixes

jasoncalabrese committed:
Merge branch 'wip/onlinestatus' of https://github.com/sulkaharo/cgm-remote-monitor into wip/onlinestatus

jasoncalabrese committed:
see if document ready is the problem when testing on travis

jasoncalabrese committed:
Revert "see if document ready is the problem when testing on travis"

This reverts commit 7428040e0536b55dff48f9b72ea783c25e5afb3c.

PieterGit committed:
Merge branch 'dev' of https://github.com/nightscout/cgm-remote-monitor into multiple_node8

philipgo committed:
Update openaps.js

philipgo committed:
Update openaps.js

philipgo committed:
Update openaps.test.js

Test for devicestatus-entries containing the correct spelling of the word "received" instead of "recieved" in OpenAPS versions 0.6.0 and later

Commit:
change node-uuid to uuid

Commit:
npm update

Commit:
add environment variables to .travis.yml

Commit:
increase timeout to 30 secs for admintools test and change environment variables for travis

Commit:
create bundle before tests and time tests execution for travis

Commit:
Makefiles requires tabs not space. I knew that...

Commit:
change Makefile

Commit:
last try for today

Commit:
set timeout of profileeditor.test to 30 seconds

Commit:
increase timeout for profileeditor tests

Commit:
only support node 8.4.0 and drop support for node 8.1.4

Commit:
bundle creation is done by npm postinstall, so should not be called explicit.

Commit:
revert tests/fixtures/headless.js

Commit:
revert test/fixtures/headless.js (now without changing indent)

Commit:
remove env's from .travis.yml (because they might not be needed)

sulkaharo committed:
Merge pull request #2773 from PieterGit/multiple_node8

Upgrade to nodejs 8.4.0 and fix unit tests for travis

sulkaharo committed:
Merge dev

sulkaharo committed:
Move all html files to views folder, update to use the rendered cache buster

sulkaharo committed:
Merge pull request #2785 from drnoname82/dev-openaps-enacted-fix

Update openaps.js to recognize the correct spelling of "received"

sulkaharo committed:
Fix tests to uses the view path :D

sulkaharo committed:
Merge dev

sulkaharo committed:
Update README to reflect correct Node version

sulkaharo committed:
Merge pull request #2689 from unsoluble/CSS-align-fix

Fix clock trend arrow alignment in iOS7/8

sulkaharo committed:
Implements #2689

sulkaharo committed:
Implements #2456

sulkaharo committed:
Merge pull request #2707 from tanja3981/alexaTranslations

Alexa translations

sulkaharo committed:
Merge dev

sulkaharo committed:
One cache buster wasn't updated to dynamic form

sulkaharo committed:
Merge pull request #2780 from nightscout/wip/onlinestatus

wip/onlinestatus + trying to fix test failures

sulkaharo committed:
Small fix to CGM data noise filtering

sulkaharo committed:
Add Node version to DEV Azure deploy json

sulkaharo committed:
Testing Node 8.1.4 for Azure

sulkaharo committed:
Testing disabling heap dumps

sulkaharo committed:
Update to 8.5.0 to see if that works without the problematic module

MilosKozak committed:
add food to websockets

MilosKozak committed:
Merge branch 'dev' of https://github.com/nightscout/cgm-remote-monitor into dev

MilosKozak committed:
add node env for deploy in azure

MilosKozak committed:
change node to 8.5.0

sulkaharo committed:
Merge pull request #2811 from MilosKozak/azure

Azure fix

sulkaharo committed:
Prepare for 0.10 release

sulkaharo committed:
Revert part of changes from previous merge

sulkaharo committed:
Merge pull request #2812 from nightscout/dev

Nightscout 0.10 release

sulkaharo committed:
Bump Nightscout version number to 0.10.1-dev

rsuvalle reviewed (commented):
Not sure why you're making hundreds of commits to my personal fork of the cgm-remote monitor. I think you mean for these commits to go to your own?

rsuvalle reviewed (commented):
Not sure how to reach out to you divishinating, but you keep trying to make commits on my person fork.

sulkaharo committed:
Fixes hourly filtering in distrubution reports, reported in #2864

unsoluble-test committed:
SVG dimension fix

Somehow the NONE.svg ended up with an incorrect rectangle shape; fixed
to match the aspect ratio of the other symbols.

MilosKozak committed:
preserve original treatment collection

unsoluble-test committed:
Remove pattern regex from basal change field

Having the regex require integers means that only the numeric keypad
will pop on iOS. This prevents the entry of negative values, which is
pretty critical for temp basals.

sulkaharo committed:
Use lodash cloneDeep instead of JSON serialize/desearialize

apanasef committed:
update Russian

sulkaharo committed:
Merge pull request #2913 from apanasef/patch-19

update Russian

sulkaharo committed:
Merge pull request #2894 from unsoluble/SVG-dimension-fix

SVG dimension fix

sulkaharo committed:
Merge pull request #2904 from unsoluble/Allow-negative-values-in-CP

Remove pattern regex from basal change field

sulkaharo committed:
Merge pull request #2911 from nightscout/fixv2apiv2

New attempt at v2 API fix

sulkaharo committed:
Cleanup: move cache buster token generation to /bin, remove /scripts

sulkaharo committed:
Fix express-minify file cache path to reduce the memory footprint of the runtime

sulkaharo committed:
Display RSSI on Loop pill

sulkaharo committed:
Add missing fontello icons to main CSS

sulkaharo committed:
Update version to release

sulkaharo committed:
Merge branch 'master' into dev

sulkaharo committed:
Loop RSSI fix

sulkaharo committed:
One more Loop RSSI fix

sulkaharo committed:
Always show RSSI

rsuvalle reviewed (commented):
Appears you're still trying to change my own personal fork of the CGM-Remote-Monitor and not your own. I get email with each one, so I'm up to over 400 emails on this so far.

sulkaharo committed:
Merge pull request #2918 from nightscout/dev

Habanero 0.10.1

sulkaharo committed:
Update dev version to 0.10.2

rsuvalle reviewed (changes requested):
Trying to get in touch with you, but you're still trying to make changes in my personal fork.

rsuvalle reviewed (changes requested):
You're using my fork again.

rsuvalle reviewed (changes requested):
You're using my fork again.

MilosKozak committed:
send foods via websockets

MilosKozak committed:
use websockeets instead of rest api for food

MilosKozak committed:
add fat,protein,energy to food editor

sulkaharo committed:
Re-enable source maps

MilosKozak committed:
Merge pull request #2924 from MilosKozak/dev

Foods update
